pkg/apiserver/registry/inventory/virtualmachine/rest.go

```diff
@@ -38,6 +38,21 @@ import (
 	"antrea.io/nephe/pkg/labels"
 )
 
+const (
+	cloudAccountName      = labels.CloudAccountName
+	cloudAccountNamespace = labels.CloudAccountNamespace
+	metaName              = "metadata.name"
+	metaNamespace         = "metadata.namespace"
+	statusCloudId         = "status.cloudId"
+	statusCloudVpcId      = "status.cloudVpcId"
+	statusRegion          = "status.region"
+)
+
+const (
+	numFieldSelectors = 5
```

Comment:
These look out of place.. Could you derive them dynamically?

Reply:
It was not needed. Removed it.